app/models/test.rb

```diff
@@ -5,7 +5,17 @@ class Test < ApplicationRecord
   has_many :users, through: :user_tests
   has_many :questions, dependent: :destroy
 
-  def self.names_by_category(name)
+  validates :title, presence: true
+  validates :title, uniqueness: { scope: :level }
+  validates :level, numericality: { greater_than_or_equal_to: 0}
```

Comment:
ТЗ: "Уровень Теста может быть только положительным целым числом"
обрати внимание на [документацию](https://guides.rubyonrails.org/active_record_validations.html#numericality) к хелперу `numericality`
> By default, it will match an optional sign followed by an integral or floating point number.

целочисленность не гарантирована по умолчанию

Reply:
Commit: 89c6582eb6c00006b1ebfd2909464be4c9f50ae2